lib/search.js

```diff
@@ -63,7 +64,11 @@ function search (args, silent, staleness, cb) {
     // prettify and print it, and then provide the raw
     // data to the cb.
     if (er || silent) return cb(er, data)
+    if (Object.keys(data).length > 10000) {
+        throw new Error("More than 10,000 matches")
+    }
     console.log(prettify(data, args))
+    console.log()
```

Comment:
`console.log` already adds newlines, so I'm wondering if the one above was hitting some line length limit? Regardless, it seems to me that this line isn't really the right way to approach that problem.